lib/app/ctx.js

```diff
@@ -224,6 +224,10 @@ Box.prototype.unwrap = function (data, content) {
             if (msg.type.substr(0, 3) === 'QLB' && tr.docs.length > 0) {
                 content = tr.docs.shift().contents;
             }
+            if (msg.type === 'DOCUMENT') {
+                data = msg.contents;
```

Comment:
What is inside `data`--ASN1 structure or another transport document?

Either way it looks like we lose everything from `tr` object, including secondary documents.